package-lock.json

```diff
@@ -34,6 +34,7 @@
       "version": "0.4.1",
       "resolved": "https://registry.npmjs.org/@cypress/listr-verbose-renderer/-/listr-verbose-renderer-0.4.1.tgz",
       "integrity": "sha1-p3SS9LEdzHxEajSz4ochr9M8ZCo=",
+      "dev": true,
```

Comment:
woah i just noticed that package-lock.json is also changed, and a HUGE amount of `"dev":true` is added, I think you probably did this by mistake as well when modifying package.json, can you revert this change as well? Having dev:true will cause these packages not being installed on production